components/dataViews/TransactionInfo.tsx

```diff
@@ -64,6 +62,19 @@ const TransactionInfo = (props: Props) => {
                   </div>
                 </li>
               </>
+            ) : msg.typeUrl === "/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward" ? (
+              <>
+                <li>
+                  <label>Amount:</label>
+                  <div>{printableCoin(props.tx.msgs[0].value.amount, state.chain)}</div>
```

Comment:
This is the same issue for the whole file, but we can do

```suggestion
                  <div>{printableCoin(msg.value.amount, state.chain)}</div>
```

since we loop over messages already. This makes the code future proof for multi message transactions (which we soon need to withdraw rewards from multiple validators).